hedera-node/test-clients/build.gradle.kts

```
@@ -258,6 +275,19 @@ val cleanValidation =
         delete(File(project.file("validation-scenarios"), "ValidationScenarios.jar"))
     }
 
+val cleanRcdiff =
+    tasks.register<Delete>("cleanRcdiff") {
+        group = "build"
+        delete(File(project.file("rcdiff"), "rcdiff.jar"))
+    }
+
+val copyRcdiff =
+    tasks.register<Copy>("copyRcdiff") {
+        group = "copy"
+        from(rcdiffJar)
+        into(project.file("rcdiff"))
```

Clearer:

```suggestion
        into(layout.projectDirectory.dir("rcdiff"))
```

I wonder if the additional _copy_ and _clean_ tasks are needed at all. The Jar task could be configured to write the Jar directly to the desired location. And when is the clean needed? (does not need to be addressed now)

---

Removed both _copy_ and _clean_ and set `destinationDirectory` in Jar.